test/jdk/javax/swing/JFileChooser/4847375/bug4847375.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 4847375 8171363
+ * @bug 4847375 8171363 8227257
```

Comment:
Could you also add an option to run this test with assertions when using jtreg directly like you did for the new test?

Reply:
Could you update the copyright year in other modified files too?